templates/docker-entrypoint.ejs

```diff
@@ -13,6 +13,23 @@ const { writeFileSync } = require('node:fs')
 <% } -%>
 <% } -%>
 
+const env = {...process.env}
+
+<% if (adonisjs) { -%>
```

Comment:
Perhaps this should be:

```js
if (adonisjs && postgres) {
```

... with the detection code for the pg node module being added to gdf.js.